### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by TymurSh**

### CODE REVIEW

**Review by NikOneZ1 — CHANGES_REQUESTED** [review]
It would be better to define skills in init method

**Comment by TymurSh** [comment]
> It would be better to define skills in init method
Isn't it necessary to create skills as a class attribute in the task conditions? I can't pass the last test when skills is an instance attributes

File: app/main.py [hunk 1]
```diff
@@ -1 +1,65 @@
-# write your code here
```
Comment (TymurSh) [inline app/main.py:1]: Isn't it necessary to create skills as a class attribute in the task conditions? I can't pass the last test when skills is an instance attributes

**Comment by TymurSh** [comment]
Isn't it necessary to create skills as a class attribute in the task conditions? I can't pass the last test when skills is an instance attributes

**Review by TymurSh — COMMENTED** [review]
Isn't it necessary to create skills as a class attribute in the task conditions? I can't pass the last test when skills is an instance attributes

**Pull request closed without merging**